nautobot/core/settings.py

```diff
@@ -479,4 +479,12 @@
         "SSL": False,
         "DEFAULT_TIMEOUT": 300,
     },
+    "custom_fields": {
```

Comment:
Need to update `nautobot/core/templates/nautobot_config.py.j2`, `development/nautobot_config.py`, and `nautobot/core/tests/nautobot_config.py` with this new queue as well.